### PR TITLE
Clean settings

### PR DESCRIPTION
**Pull request by eugene-babichenko**
Implements issue REM-1352.

Removal of settings from batch injector https://github.com/Remmeauth/sawtooth-core/pull/9

### CODE REVIEW

File: remme/tp/consensus_account.py [hunk 1]
```diff
@@ -52,6 +51,8 @@
 FAMILY_NAME = 'consensus_account'
 FAMILY_VERSIONS = ['0.1']
 
+UNFREEZE_BONUS = Decimal("10")
```
Comment (DeV1doR) [inline remme/tp/consensus_account.py]: Move to the settings file better

File: remme/tp/consensus_account.py [hunk 2]
```diff
@@ -145,9 +146,9 @@ def _send_reward(self, context, public_key, pb_payload):
             ZERO_ADDRESS: zero_account,
         }
 
-        if unfreeze_bonus <= reputational < min_stake * unfreeze_bonus:
-            unfrozen_share = self._calculate_share(max_share, min_share, unfreeze_bonus,
-                                                   min_stake, reputational)
+        if UNFREEZE_BONUS <= reputational < min_stake * UNFREEZE_BONUS:
```
Comment (DeV1doR) [inline remme/tp/consensus_account.py]: Here are a mistake and need o change from UNFREEZE_BONUS to min_stake (anyway it will be in another PR)

File: remme/tp/consensus_account.py (hunk 1 above)
Comment (eugene-babichenko) [reply remme/tp/consensus_account.py]: Don't see any reason to move a constant that is used only in one file to another file.

Comment (DeV1doR) [reply remme/tp/consensus_account.py]: Because of it possibly could be used at another place + one place of constants are better to lookup and change it faster. IMXO

Comment (ArtemFrantsiian) [reply remme/tp/consensus_account.py]: We don't need to change this value during the work process
